release_notes/41-release-notes.adoc

```diff
@@ -409,9 +410,21 @@ When Central was reinstalled, the password of the new secret didn't match the on
 
 Release date: 11 July 2023
 
-* Previously, upgrading from RHACS 4.0.2 to RHACS 4.1 using the RHACS Operator would fail because of the deprecated KernelModule collector support in RHACS 4.1.
+* Previously, upgrading from {product-title-short} 4.0.2 to {product-title-short} 4.1 using the {product-title-short} Operator would fail because of the deprecated KernelModule collector support in {product-title-short} 4.1.
 The updated image fixes this issue.
 
+[id="resolved-in-version-412_{context}"]
+=== Resolved in version 4.1.2
+
+Release date: 26 July 2023
+
+* {product-title-short} 4.1.2 includes a fix for an issue where an operator-based {product-title-short} installation failed to upgrade from versions earlier than {product-title-short} 3.71.
+The failure occurred because the older versions did not implement PSP auto-sensing on the Kubernetes cluster 1.25 or later.
+* Previously, if you were using {product-title-short} with Istio, the Scanner pods would crash if Istio reported the version as `latest` instead of the actual version number, such as `1.17`.
+The updated image fixes this issue.
+* Fixed an issue where upgrading from RHACS 3.74.3 to RHACS 4.1.1 failed
+with an "insert...violates foreign key restraint" error.
```

Comment:
@lahinson Can you review these additional lines I've added after your review? Thanks.

Reply:
Patch is released so I'm merging this now. We can do a follow up PR if there are changes needed.